docs/general/builtins/scalar-functions.rst

```diff
@@ -3159,6 +3159,41 @@ arguments, the return type is ``numeric``.
     SELECT 1 row in set (... sec)
 
 
+.. _scalar-array_unnest:
+
+``array_unnest(nested_array)``
+------------------------------
+
+Takes a nested array and returns a flattened array. Only flattens one level at a
+time.
+
+Returns ``NULL`` if the argument is ``NULL``. If the argument is an array that
+contains ``NULL`` elements, the elements are skipped.
+
+
+::
+
+    cr> SELECT array_unnest([[1, 2], [3, 4, 5]]) AS result;
+    +-----------------+
+    | result          |
+    +-----------------+
+    | [1, 2, 3, 4, 5] |
+    +-----------------+
+    SELECT 1 row in set (... sec)
+
+    cr> SELECT array_unnest([[1, 2], null, [3, 4, 5], null]) AS result;
+    +-----------------+
```

Comment:
Can we modify `[3, 4, 5]` to `[3, 4, null, 5]` to highlight that only null-arrays are skipped but null-elements are not?

Reply:
maybe even a 2 level nesting to show the  `one level at a time` ?

Reply:
Added a `null` leaf and rephrased the text a bit. I didn't include 2 level nesting. I think two examples should be enough for this function